### ISSUE TITLE
broken mdi icons

### ISSUE THREAD

**Issue by expilo** [lumapps/lumX#442]
Hello, 
Out of a sudden all of my mdi-icons changed in my website. For instance instead of facebook-box I'm getting chevron-double-down.  I'm using lumx 1.0.5 and mdi 1.6.50. Any ideas?

**Comment by matmkian** [comment]
I have the same versions and no problem for me... How do you display your icons? With lumX icon directive?

**Comment by expilo** [comment]
`<li class="list-row list-row--has-separator">
											<div class="list-row__primary">
												<span class="icon icon--s icon--grey icon--flat mdi mdi-numeric-1-box-multiple-outline"></span> 
											</div>
											<div class="list-row__content">
												<span>Some text...</span>
											</div>
										</li>`

**Comment by expilo** [comment]
If I use the directive the same problem persists.

**Comment by afonsof** [comment]
I have the same problem. Receiving this error in stacktrace:
```
Error: Undefined variable: "$mdi-icons".
        on line 51 of bower_components/lumx/core/scss/tools/_mixins.scss, in `mdi-icon'
        from line 87 of bower_components/lumx/modules/checkbox/scss/_checkbox.scss
        from line 41 of bower_components/lumx/core/scss/_lumx.scss
```

**Comment by matmkian** [comment]
Do you still have the issue? I know that the author of material design icons has recently changed the name of his font... @expilo Do you use Sass version or simply include lumx.css in your index head? @afonsof Have you tried to clean your bower cache and update bower?

**Comment by expilo** [comment]
Yes, I'm using Sass version.

2016-06-13 8:58 GMT+02:00 matmkian <notifications@github.com>:

> Do you still have the issue? I know that the author of material design
> icons has recently changed the name of his font... @expilo
> <https://github.com/expilo> Do you use Sass version or simply include
> lumx.css in your index head? @afonsof <https://github.com/afonsof> Have
> you tried to clean your bower cache and update bower?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lumapps/lumX/issues/442#issuecomment-225502421>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAAEfTFQrooehOJmytGr9QuvCSaar-dGks5qLP-CgaJpZM4IjwCg>
> .
>



-- 
http://okle.pl

**Comment by afonsof** [comment]
My problem was solved after updating to version 1.0.6
